community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/Tracker.java

```diff
@@ -63,4 +63,7 @@
      * @param value value to set at that index.
      */
     void set( long index, long value );
+
+    @Override
+    void close();
```

Comment:
It could, I just saw no reason for involving an interface from `org.neo4j.graphdb` into this.